src/main/java/org/folio/gobi/Mapper.java

```diff
@@ -87,17 +67,17 @@ public Mapper(Map<Mapping.Field, DataSourceResolver> mappings) {
       mapSource(futures, source, doc);
       mapContributor(futures, contributor, doc);
       mapReportingCodes(futures, reportingCode, doc);
-      mapMetaData(futures, metaData, doc);
+      mapVendorDependentFields(futures,eresource,physical,compPO, claim, doc);
```

Comment:
Nitpick... please use a space between comma-separated values... e.g. `foo, bar, baz`  instead of `foo,bar,baz`